pyfarm/master/api/agents.py

```diff
@@ -620,9 +632,11 @@ def post(self, agent_id):
 
         current_assignments = g.json.pop("current_assignments", None)
         mac_addresses = g.json.pop("mac_addresses", None)
+
         # TODO return BAD_REQUEST on bad mac addresses
         if mac_addresses is not None:
-            mac_addresses = [x.lower() for x in mac_addresses if MAC_RE.match(x)]
+            mac_addresses = [
+                x.lower() for x in mac_addresses if MAC_RE.match(x)]
```

Comment:
This is a purely cosmetic change that shouldn't be mixed in with a major functionality change.